CHANGELOG.md

```diff
@@ -1,3 +1,65 @@
+# [v0.1.1](https://github.com/n0-computer/iroh/compare/v0.1.0...v0.1.1) (2022-11-21)
+
+## Bitswap client mode
+Iroh can now run in "client mode", which can _fetch_ content from bitswap, but not _provide_ content. This is useful if you're running on a lower-powered device or have limited bandwidth. To use client mode change your `p2p.config.toml`:
+```
+bitswap_server = false
+```
+
+## Use indexer nodes to find content providers
+Iroh can use network indexers like [cid.contact](https://cid.contact) to find providers of a given CID. To use the cid.contact indexer adjust your `gateway.config.toml` to add an `indexer_endpoint` URL:
+
+```
+indexer_endpoint = https://cid.contact/cid/
+```
+
+## Docker support
+Iroh now ships with docker images! Spin iroh up locally with:
+```
+$ cd iroh/docker
+$ docker-compose up
+```
+then run `iroh status` from another terminal if you already have iroh installed. Now you're backing iroh with docker! `docker-compose down` to stop. The docker compose file also includes helpful guidance on which ports to expose from each iroh service.
+
+We're shipping multi-arch [distroless](https://github.com/GoogleContainerTools/distroless) builds for amd64 & arm64 architectures.
+
+## Fixed a critical bug adding files
+We fixed a [known bug](https://github.com/n0-computer/iroh/issues/423) in iroh v0.1.0 that would break `iroh add` if files referenced the same CID on the way in. It's now fixed, and now we can happily do roundtrip add/get of the linux kernel, which is always fun 😊.
+
+
+## Benchmarks
+Benchmarks for this release coming soon. We'll update this readme once they're up.
```

Comment:
"this readme" -> "this changelog"?

not critical

Reply:
yeah, I'll make a point of updating later